docs/docs/user/migration-guide/v1-v2.md

```diff
@@ -18,7 +18,7 @@ Because of this new convention `producer` has been renamed to `producer-app`. Th
     app:
         streams:
         outputTopic: output_topic
-        extraOutputTopics: 
```

Comment:
Why can't `dprint` not format trailing spaces?

Reply:
it's inside a code block (and there is no formatter for the `diff` syntax)

Reply:
you could set this up in your editor (auto trim whitespace on save)